engine/tests/test_lua10_lust_eval.lua

```diff
@@ -0,0 +1,86 @@
+local cloe = require("cloe")
+local project = cloe.require("project")
+local lust = require 'lust'
+local it, expect = lust.it, lust.expect
+
+project.init_report()
+
+cloe.apply_stack {
+    version = "4",
+    include = {
+        "config_nop_infinite.json",
+    },
+    server = {
+        listen = false,
+        listen_port = 23456,
+    },
+}
+
+--cloe.init_report()
+
+-- All the conditions we want to fail on:
+cloe.schedule_these {
```

Comment:
- [ ] Remove unnecessary statements from test

We want to try to keep tests focused. In this test, lets remove all the stuff we don't need, keep it minimal and focused on the thing we are trying to demonstrate and test.